noteable_magics/sql/magic.py

```diff
@@ -35,7 +35,7 @@ class SqlMagic(Magics, Configurable):
     Provides the %%sql magic."""
 
     short_errors = Bool(
-        True,
+        False,
         config=True,
         help="Don't display the full traceback on SQL Programming Error",
     )
```

Comment:
This should not block this PR _at all_ but I do think it's worth including some metadata that Planar Ally can process that will be quiet if the feature flag isn't set.